x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/stats/SearchStats.java

```diff
@@ -51,6 +51,7 @@ private static class FieldStat {
         private Boolean singleValue;
         private Boolean hasIdenticalDelegate;
         private Boolean indexed;
+        private Boolean hasDocValues;
```

Comment:
looks like the naming convention is to have no has prefix so `docValues`

Reply:
Or add `is` and `has` prefixes to the rest... (which might conform better to the naming contexts elsewhere).